pkg/kubelet/cm/cpumanager/policy_static_test.go

```diff
@@ -18,6 +18,9 @@ package cpumanager
 
 import (
 	"fmt"
+	"github.com/blang/semver/v4"
+	"k8s.io/component-base/metrics/testutil"
+	"k8s.io/kubernetes/pkg/kubelet/metrics"
```

Comment:
misplaced imports, please place them like this:
- stdlib imports
- 3rd party non-kubernetes imports
- kubernetes imports

Reply:
Ah, got it ~